pages/index.js

```diff
@@ -78,22 +78,28 @@ export default function Home() {
                 )}
                 {/* ACTION */}
                 <Link href="/account-verification" passHref>
-                  <Button as="a" variant="inverted" block>
+                  <Button as="a" variant="bold" block>
                     {basiqConnectionInProgress || basiqConnectionSuccess ? 'Continue setup' : 'Get started'}
                   </Button>
                 </Link>
               </div>
             </div>
           )}
         </div>
-
-        {/* Decorative blurry circles */}
-        <div className="absolute w-full h-screen overflow-hidden">
-          <div className="absolute -mt-64 bg-white rounded-full opacity-40 w-80 h-80 top-1/2 left-1/2 -ml-[800px] blur-xl mix-blend-overlay" />
-          <div className="absolute -mt-10 bg-white rounded-full opacity-50 w-52 h-52 top-1/2 left-1/2 -ml-80 blur-md mix-blend-overlay" />
-          <div className="absolute -ml-16 bg-white rounded-full -mt-[550px] opacity-10 w-72 h-72 top-1/2 left-1/2 blur-2xl mix-blend-overlay" />
-          <div className="absolute w-32 h-32 ml-24 -mt-32 bg-white rounded-full opacity-25 top-1/2 left-1/2 blur-sm mix-blend-overlay" />
-          <div className="absolute mt-24 bg-white rounded-full opacity-40 w-96 h-96 top-1/2 left-1/2 ml-[450px] blur-xl mix-blend-overlay" />
+        {/* Decorative blurry circles*/} 
+        <div className="absolute w-full h-screen">
+          <div className="absolute hidden w-[243px] h-[243px] top-16 left-48 sm:block">
+            <img src="/planet-top-left.svg" alt="planet1" />
+          </div>
+          <div className="absolute right-0 sm:w-[297px] sm:h-[347px] sm:-top-4 sm:right-0 w-[227px] h-[257px]">
+            <img src="/planet-top-right.svg" alt="planet2"/>
+          </div>
+          <div className="absolute sm:w-[500px] sm:h-[500px] sm:bottom-12 sm:left-0 w-[200px] h-[200px] bottom-4 -left-16">
+            <img src="/planet-bottom-left.svg" alt="planet3"/>
+          </div>
+          <div className="absolute hidden w-[256px] h-[256px] bottom-28 right-48 -rotate-[15deg] sm:block">
+            <img src="/planet-bottom-right.svg" alt="planet3"/>
+          </div>
```

Comment:
This block repeats in AccountVerificationForm.js also. Can you put it in another component and reuse ?